src/org/opendatakit/briefcase/ui/export/ExportPanelForm.java

```diff
@@ -120,6 +133,22 @@ private void setExporting(boolean active) {
     exporting = active;
   }
 
+  synchronized public void updateExportingLabel() {
+    String text = lblExporting.getText();
+    if (text.equals(EXPORTING_DOT_ETC)) {
+      text = "Exporting.";
+    } else {
+      text += ".";
+    }
+    lblExporting.setText(text);
+
+    //set a timeout of 5 secs and hide label if there are no new updates
+    scheduledHideExportingLabel.ifPresent(scheduledFuture ->
+            scheduledFuture.cancel(false));
+    scheduledHideExportingLabel = Optional.of(SCHEDULED_EXECUTOR.schedule(this::hideExporting,
+            5, TimeUnit.SECONDS));
```

Comment:
Also, `import static` the `TimeUnit.SECONDS` member to be more concise. This line should read: 

```java
scheduledHideExportingLabel = Optional.of(SCHEDULED_EXECUTOR.schedule(this::hideExporting, 5, SECONDS);
```